### PR TITLE
tbi: telemetry settings store

### PR DESCRIPTION
**Pull request by slin-sap**
For TBI https://github.com/SAP/open-ux-tools/issues/358

Temporary implement of telemetry setting config using existing store framework so it doesn't block the main central telemetry config user story. Will move the telemetry specific implementation out once the store module is refactored.

### CODE REVIEW

File: packages/store/src/entities/telemetry-setting.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { EntityKey } from '.';
+import { serializable } from '../decorators';
+
+/** This represents the migration status of the systems from pure secure store
```
Comment (tobiasqueck) [inline packages/store/src/entities/telemetry-setting.ts:4]: I don't understand this comment

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
Link to actual issue is missing (I am assuming it is #358) and neither issue nor PR contain sufficient information to understand why this is done.

**Review by ullasholla — APPROVED** [review]
Thanks @slin-sap, looks good. Test coverage is good too.

**Review by tobiasqueck — APPROVED** [review]
The code is simple and well structured. I am missing comments for each function, however, since the functions are self-explanatory I am not seeing it as a reason to block the merge.

**Pull request merged**